### PR TITLE
fix English doc for dice_loss, log_loss, unfold and NLLLoss API test=…

### PR DESCRIPTION
**Pull request by lijianshe02**
### PR types

Breaking changes
### PR changes

Docs
### Describe

update 2.0 doc for dice_loss, log_loss, unfold, NLLLoss

dice_en
log_en
unfold_en

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -7108,7 +7108,6 @@ def dice_loss(input, label, epsilon=0.00001, name=None):
             import paddle
             import paddle.nn.functional as F
 
-            paddle.disable_static()
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py:7111]: 7073~7075 alias的内容需要删除；
其他地方的同理

File: python/paddle/nn/layer/loss.py [hunk 2]
```diff
@@ -703,8 +703,6 @@ class NLLLoss(fluid.dygraph.Layer):
                                  [0.05689114, 0.0862954 , 0.6325046 ]]).astype(np.float32)
                 label_np = np.array([0, 2, 1, 1, 0]).astype(np.int64)
 
-                place = paddle.CPUPlace()
-                paddle.disable_static(place)
                 input = paddle.to_tensor(input_np)
```
Comment (TCChenlong) [inline python/paddle/nn/layer/loss.py:704]: 696行 改为 paddle.nn.NLLLoss
710行print(result) 即可

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]
Please upgrade the dygraph branch of `paddle.fluid.layers.unfold` and `paddle.fluid.layers.log_loss` in future.

**Review by chalsliu — APPROVED** [review]
